.github/workflows/ci.yml

```
@@ -110,9 +110,19 @@ jobs:
           - scenario: on ubuntu
             runs-on: ubuntu-24.04
             os: linux
+            experimental: false
```

```suggestion
            experimental-installer: false
```

It may seem a bit redundant, but experimental has other meanings and it's worth being explicit here.